tox.ini

```diff
@@ -19,3 +19,28 @@ deps =
     yamllint==1.15.0
 commands =
     bash -c "yamllint eve/main.yml salt/**/*.yaml"
+
+[testenv:ci-tests]
+deps =
+    pytest
```

Comment:
Let's have a `tests/requirements.txt` and `-r` that one.

Reply:
and fixed versions for these requirements :)

Reply:
In MetalK8s 1.x, we have a couple of `requirements.in` files which specify the reqs with only lower bounds (and maybe next-major-upper bounds? Can't remember) and a `tox` rule to `pip-compile` those into the various `requirements.txt` files. Makes it trivial to update to newer versions, even when only a single package needs to be updated (e.g. because of security issues).